nxsdk_modules_contrib/pelenet/lib/anisotropic/connection_matrix.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# connection_matrix.py
+#
+# Copyright 2017 Sebastian Spreizer
+# The MIT License
```

Comment:
Remove copyright and license. All contributed modules in this repository are covered by a master license file.

Reply:
The code in the /lib/anisotropic folder is not from me, but from another scientist and I would like to mention this somewhere: Where would be the best place to do this?

Reply:
In the README under Acknowledgements, you can mention this way:

- **Pelenet : Carlo Michaelis**
   - Connection matrix and Connection asymmetry landscape Python packages were contributed by **Sebastian Spreizer**